trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
```

Comment:
- [ ] When testing your application I received the below error due to the use of `require` instead of `require_relative` please make sure to use the second option in order to make your application work
![afbeelding](https://user-images.githubusercontent.com/39240820/233018956-371275f6-aeee-4c08-b6e3-2aeaf8730248.png)